src/Layer/OrientedImageLayer.js

```diff
@@ -0,0 +1,166 @@
+import * as THREE from 'three';
+import GeometryLayer from 'Layer/GeometryLayer';
+import OrientedImageMaterial from 'Renderer/OrientedImageMaterial';
+import ImmersiveControls from 'Controls/ImmersiveControls';
+import GeoJsonParser from 'Parser/GeoJsonParser';
+import CameraCalibrationParser from 'Parser/CameraCalibrationParser';
+import Coordinates from 'Core/Geographic/Coordinates';
+import OrientationUtils from 'Utils/OrientationUtils';
+
+const coord = new Coordinates('EPSG:4978', 0, 0, 0);
+
+function updatePano(context, camera, layer) {
+    // look for the closest oriented image
+    let minD = Infinity;
+    let minI = -1;
+    for (let i = 0; i < layer.panos.length; i++) {
+        const position = layer.panos[i].position;
+        const D = camera.position.distanceTo(position);
+        if (D < minD) {
+            minD = D;
+            minI = i;
+        }
+    }
+    // detection of oriented image change
+    const ori = layer.panos[minI];
+    if (ori && layer.currentPano != ori) {
```

Comment:
Could you uniformize the name `ori` and `pano`?
To avoid having multiple denomination for the same concept.